execution/executor-types/src/lib.rs

```diff
@@ -44,31 +44,37 @@ pub struct StateComputeResult {
     /// for StateMachineReplication, which is merely passing it between StateComputer and
     /// TxnManager.
     compute_status: Vec<TransactionStatus>,
+    /// The transaction info hashes of all success txns.
+    transaction_info_hashes: Vec<HashValue>,
 }
 
 impl StateComputeResult {
     pub fn new(
-        accu_root_hash: HashValue,
```

Comment:
why you make this type of critical change here? this is part of the refactoring you could have done in another PR :P

Reply:
Yes, but this one seems trivial. The separation of it doesn't help a lot.